src/xo-app/host/tab-network.js

```diff
@@ -189,21 +280,20 @@ class PifItem extends Component {
             className={pif.carrier ? 'text-success' : 'text-muted'}
           />
         </Tooltip>
-      </td>
-      <td className='text-xs-right'>
-        <ActionRowButton
-          disabled={pif.physical || pif.disallowUnplug || pif.management}
-          handler={deletePif}
-          handlerParam={pif}
-          icon='delete'
-          tooltip={_('deletePif')}
-        />
-      </td>
-    </tr>
+      </div>
+    },
+    name: _('pifStatusLabel')
   }
-}
+]
+const INDIVIDUAL_ACTIONS = [
+  {
+    handler: deletePif,
+    icon: 'delete',
+    label: _('deletePif')
+  }
+]
 
-export default ({
+export default({
```

Comment:
Why this change?

Reply:
Because of a Linter message i think.

Reply:
Can you tell me which error did the linter report here?

Because it looks incorrect.

Reply:
The current version contains a stateful component instead.